spec.go

```diff
@@ -53,6 +57,8 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
+	// ExtraFiles specifies open file descriptors that should be passed to the process.
+	ExtraFiles []os.File `json:"extraFiles"`
```

Comment:
@crosbymichael Systemd socket activated containers http://0pointer.de/blog/projects/socket-activated-containers.html
We could either make this generic or leave it out and have the runtimes support systemd environment variables for activation.